people_detector/labeled_obj[] lb_objs_vector
